### PR TITLE
fix: double scrollbar in details pane

### LINKED ISSUE #3094: 2 scroll bars when content is bigger than the page
### Describe the bug

Screenshot from 2022-11-08 15-56-00

### Steps to reproduce

1. Open contacts
2. click on a contact that has too many fields or make the screen smaller
3. See a scroll bar poping up on the right

### Expected behavior

1 scroll bar

### Actual behavior

2 scroll bars are shown

### Contact version

_No response_

### Operating system

_No response_

### PHP engine version

_No response_

### Web server

_No response_

### Database

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by st3iny**
Fix #3094

### CODE REVIEW

File: src/components/ContactDetails.vue [hunk 1]
```diff
@@ -832,13 +832,6 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-// Container height fix for dropdowns
-.app-content-details {
-	height: 100%;
-	min-height: calc(100vh - var(--header-height));
```
Comment (szaimen) [inline src/components/ContactDetails.vue:838]: Are you sure that this does not break scrolling on mobile?

Comment (st3iny) [reply src/components/ContactDetails.vue:838]: I tested it and it worked fine. The parent containers (from nc-vue) make sure of that.

Comment (szaimen) [reply src/components/ContactDetails.vue:838]: all right, cool!

**Review by szaimen — APPROVED** [review]
LGTM but didnt test

**Pull request merged**